lib/postgrex/pgpass.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Postgrex.Pgpass do
+  @moduledoc """
+  Parses .pgpass file to obtain database credentials
+  """
+
+  @pgpass_path Application.get_env(:postgrex, :pgpass, Path.join(System.user_home, '.pgpass'))
```

Comment:
Additionally this won't be correct if you compile code in one place and than move over to another machine (or just change the user).